packages/iota/src/utils/validationHelper.ts

```diff
@@ -0,0 +1,62 @@
+// Copyright 2020 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+import type { AddressTypes } from "../models/addresses/addressTypes";
+import type { IRent } from "../models/IRent";
+import { BASIC_OUTPUT_TYPE, IBasicOutput } from "../models/outputs/IBasicOutput";
+import { ADDRESS_UNLOCK_CONDITION_TYPE } from "../models/unlockConditions/IAddressUnlockCondition";
+import type { IValidationResult } from "../validation/result";
+import { TransactionHelper } from "./transactionHelper";
+
+/**
+ * Class to help with validation.
+ */
+export class ValidationHelper {
+
+    /**
+     * Validate that the values in Set are unique.
+     * @param elements The elements.
+     * @param targetSize The number of decimal places to display.
```

Comment:
On second thought, if we change it to take the "array of types", we don't even need this "targetSize", which simplifies the usage.